Library/Homebrew/cask/installer.rb

```diff
@@ -53,6 +53,27 @@ def self.print_caveats(cask)
       puts caveats + "\n"
     end
 
+    def self.print_gatekeeper_warning(cask)
+      ohai "Gatekeeper warning"
+      puts <<~EOS
+        Your Cask has been placed under the supervision of macOS's Gatekeeper.
+        This grants Casks the same level of protection as apps you've installed
+        manually; however, they will be lauched from an unspecified, read-only
+        path. This is called "path randomization" or "App Translocation".
+
+        If this causes you problems, please move the app yourself using the
+        Finder, or reinstall using:
+
+          brew cask reinstall #{cask} --no-quarantine
+
+        For more information, please read the following resources:
+
+          https://www.apple.com/business/resources/docs/macOS_Security_Overview.pdf, section "App Security", "Gatekeeper"
+
+          https://www.synack.com/2016/12/16/untranslocating-apps/
+      EOS
+    end
+
```

Comment:
Is the warning still necessary or helpful to the average user?

Reply:
In light of your translocation bits, no. Will remove.